packages/app/src/domain/tested/infected-per-age-group/infected-per-age-group.tsx

```diff
@@ -51,43 +48,67 @@ export function InfectedPerAgeGroup({
     });
   }, [text.legend]);
 
+  const underReported = useMemo(
```

Comment:
```suggestion
  const underReportedConfigItem = useMemo(
```

It wasn't immediately clear to me what this variable contained.